lib/routes/51cto/recommend.ts

```diff
@@ -2,6 +2,11 @@ import { Route } from '@/types';
 import { parseDate } from '@/utils/parse-date';
 import got from '@/utils/got';
 import { getToken, sign } from './utils';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import { ofetch } from 'ofetch';
```

Comment:
```suggestion
import ofetch from '@/utils/ofetch';
```
https://docs.rsshub.app/joinus/new-rss/start-code#basic-code